release-notes/release-notes59.md

```diff
@@ -131,7 +131,15 @@ For information on the Table plugin, see: [Table plugin]({{site.baseurl}}/plugin
 
 The following enhancements were made for the {{site.productname}} 5.9 release.
 
-### Enhancement name
+## Context Toolbars
```

Comment:
```suggestion
### Context Toolbars
```
Should it be this way around?

Reply:
Tyler has already addressed this in his own suggestion, that adds more improvements.